docs/source/release/v6.12.0/Direct_Geometry/MSlice/New_features/38299.rst

```diff
@@ -0,0 +1 @@
+- MSlice is now installed via Conda as part of Mantid workbench and not compiled as an external project anymore.
```

Comment:
Will a user experience any different behaviour after this change? i.e do we need a release note detailing the change.

Reply:
The user should not see any difference in behaviour to before. 

For developers, there are two differences: when using the mantid-developer package, they will see a warning when clicking on `MSlice` in the menu instead of the interface. Since MSlice is not developed through the mantid repo, but has its own repo, most developers will not even notice this. The other change is that MSlice is not downloaded from the repo after SHA changes (there won't be SHA changes anymore).